lib/coxir/struct/user.ex

```diff
@@ -102,6 +102,23 @@ defmodule Coxir.Struct.User do
     |> Channel.pretty
   end
 
+  @doc """
+  Sends a DM to a given user.
+
+  Returns a message object upon success
+  or a map containing error information.
+  """
+  @spec send_message(user, String.t) :: map
+
+  def send_message(%{id: id}, content),
+    do: send_message(id, content)
+
+  def send_message(recipient, content) do
+    create_dm(recipient)
+    |> Channel.send_message(content)
+    |> Channel.pretty
```

Comment:
This is wrong. The function on top returns a [prettified message](https://github.com/satom99/coxir/blob/fca39acc18ac76363a8033cdabea505cfb49b161/lib/coxir/struct/channel.ex#L69) already.

So tldr; remove that line.